src/main/java/com/cs56fitnessapp/models/workout/StrengthTraining.java

```diff
@@ -0,0 +1,90 @@
+package com.cs56fitnessapp.models.workout;
+
+import com.cs56fitnessapp.models.User;
+import java.time.LocalDate;
+
+/**
+ * @author Daniel Cervantes
+ * @author Ksenia Koldaeva
+ * Date created: 10/18/17
+ * Last updated: 11/11/17
+ */
+
+public class StrengthTraining extends Workout {
+
+    private StrengthTrainingLevel strengthTrainingLevel;
+    private boolean training;
+
```

Comment:
Currently, I can not see an appropriate use-case for using data field `training`.
It looks potentially useful, when there is a timer functionality - e.g. select start time, and while the timer runs you can provided the user with calories spent in real time. 

For now, I think, it might be logically confusing. See below under `getCaloriesOut` review.
I would suggest to remove it.